tests/component/task/test_task_read_waveform_di.py

```diff
@@ -61,28 +71,26 @@ def test___digital_single_channel___read_waveform_lines_and_port___returns_valid
 
     waveform = di_single_chan_lines_and_port_task.read_waveform(samples_to_read)
 
-    # Note, the data on the port's waveform is MSB instead of LSB because of bug AB#3178052
-    # When that bug is fixed, these asserts should be updated
-    assert _get_waveform_data(waveform) == [0, 1025, 514, 1539, 260, 1285, 774, 1799, 128, 1153]
+    assert _get_waveform_bitstrings(waveform) == [
+        "00000000000",
```

Comment:
Why not use binary integer literals? Do the bitstrings produce better failure messages?

```python
>>> 0b10000
16
```